server/events.go

```diff
@@ -206,6 +206,7 @@ type AccountNumConns struct {
 // AccountStat contains the data common between AccountNumConns and AccountStatz
 type AccountStat struct {
 	Account       string    `json:"acc"`
+	AccountName   string    `json:"acc_name"`
```

Comment:
Should this be just name? Should it be omit empty?

Reply:
Changed to "name". I checked that within the other embedded fields, there isn't a conflict and it should be contextualized enough to not be ambiguous.

Reply:
Still seems to be AccountName